### PR TITLE
Rackspace SDK changed AGAIN

### PR DESCRIPTION
**Pull request by ghost**
There was a change to the rackspace SDK.  This broke this package.  I updated the stream wrapper unit test.  It was creating a mock of the RSCService object, the unit test was supposed to test this service... The test now correctly exercises the RSCService Object and uses mock objects for it's dependecies.  All tests were green.

### CODE REVIEW

**Comment by liuggio** [comment]
Changed again?
Thanks a lot!
@toretto460 told me that will look into that soon

**Comment by ghost** [comment]
@liuggio Your Welcome... I need this ASAP so I can deploy code again! lol... Please review it before you merge it... I did something wrong because it's trying to include your previous commit too.

**Comment by claudio-dalicandro** [comment]
https://github.com/rackspace/php-opencloud/issues/214
The state of the SDK does not seem to be stable at the moment...
take a look at the DataObject::__construct()...
https://github.com/rackspace/php-opencloud/commit/225f04a316c409e27b7295ccb3d2d5ee1d32c077

Edit: If you need to use the StreamWrapper use the php-opencolud commit referenced into the composer.lock!

File: src/Tvision/RackspaceCloudFilesStreamWrapper/Service/RSCFService.php [hunk 1]
```diff
@@ -112,8 +139,9 @@ public function apiGetObjectByContainer(Container $container, $objectData)
         if (!$container) {
             return false;
         }
-        $object = $container->DataObject();
-        $object->name = $objectData['name'];
+
+        $object = $container->dataObject();
+        $object->setName($objectData['name']);
```
Comment (claudio-dalicandro) [inline src/Tvision/RackspaceCloudFilesStreamWrapper/Service/RSCFService.php:144]: This method don't exists at the moment... But the property name it's now protected... Please wait the fix!

**Comment by ghost** [comment]
I found the issue it was in the composer.json file... Should have been version 1.7.\* not 1.6.*...  But have other issues I have to track down... Realizing the unit tests are mocking the objects they are supposed to be testing, which is a code smell for me... Looking into that so the tests will exercise the objects correctly.

**Comment by claudio-dalicandro** [comment]
@cdwlighting the streamWrapper now uses php-openCloud 1.7. \* , there is no more RackspaceAPI class, the current version directly instantiate an ObjectStore using a static factory... Let me know if you need help!

**Pull request closed without merging**